projects/remove-all-whitespace/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Remove All Whitespace in Every Language
+layout: default
+date: 2020-10-21
+last-modified: 2020-10-24
+featured-image:
+tags: [remove-all-whitespace, c]
+authors:
+    - barhouum7
+---
+
+In this article, we'll outline the Remove All Whitespace project. 
+
+## Requirements
+
+A string is a collection of characters. Sometimes strings contain whitespace characters like " ", "\t", and "\n". 
+The purpose of this program is to remove all such spaces from a string as follows:
```

Comment:
Make sure to pull from master. This file should already be in there.

Reply:
Yeah! It's all right